Assets/MixedRealityToolkit/Utilities/MRTKServicesUtils.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Boundary;
+using Microsoft.MixedReality.Toolkit.Diagnostics;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.Teleport;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// Utility class to easily access references to runtime Mixed Reality Toolkit Services
+    /// </summary>
+    public static class MRTKServicesUtils
```

Comment:
Naming nit: Consider calling this utility `Services`?

`Toolkit.MRTKServiceUtils.InputSystem` reads like some kind of hack or workaround.

`Toolkit.Services.InputSystem` reads more like an officially sanctioned approach.

Reply:
Or, ServiceCache?

Reply:
ok I'll rename to ServiceCache and update comments to note that referenced services may be swapped out and no longer available if user modified at runtime